agent.py

```diff
@@ -51,8 +51,7 @@ def __init__(self, agentID, birthday, cell, configuration):
         self.cellsInRange = []
         self.lastMoved = -1
         self.neighborhood = []
-        self.vonNeumannNeighbors = {"north": None, "south": None, "east": None, "west": None}
-        self.mooreNeighbors = {"north": None, "northeast": None, "northwest": None, "south": None, "southeast": None, "southwest": None, "east": None, "west": None}
+        self.socialNeighbors = []
```

Comment:
Let's just call this `self.neighbors`.

Reply:
Especially since we have the `updateNeighbors` method for agents.

Reply:
Renamed in new commit.